plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -45,6 +45,8 @@
     // columns of the relation described by this handle
     private final Optional<List<JdbcColumnHandle>> columns;
 
+    private final int syntheticNextIdentifier;
```

Comment:
`nextSyntheticColumnId` or `nextSyntheticNextIdentifier`?

Reply:
i picked the existing name from the method in JdbcMetadata, but i don't feel strongly. Please advise?

Reply:
changed to `nextSyntheticColumnId`